<?xml version="1.0"?>
<launch>
    <arg name="nDrones" default="12" />
    <arg name="obstacleConfig" default="obstacles.yaml"/>
    <arg name="startsConfigName" default="starts.txt"/>
    <arg name="data_dir" default="$(find prm_vis)/launch/data/"/>

    <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find prm_vis)/launch/custom_rosconsole.conf"/>
  
    <node name="prm_vis" pkg="prm_vis" type="prm_vis" output="screen">
        <param name="data_dir" value="$(arg data_dir)"/>
        <param name="nDrones" value="$(arg nDrones)"/>
        <param name="obstacleFileName" value="$(arg obstacleConfig)"/>
        <param name="startsConfigName" value="$(arg startsConfigName)"/>

    </node>

      <!-- <node name="rviz"
        pkg="rviz"
        type="rviz"
        output="screen"
        args="-d $(find prm_vis)/launch/simulation.rviz"
        /> -->
  />

</launch>
